data/structures.yml

```diff
@@ -1015,7 +1018,10 @@ NZ:
 US:
   :bank_code_length: 9
   :branch_code_length: 0
-  :account_number_length: 17
+  :account_number_length: !ruby/range
+    begin: 1
```

Comment:
Is it really possible to have a one-digit account number?

Reply:
I'm not too sure, I'd hope not!